SocketIOClientSwift/SocketIOClient.swift

```diff
@@ -425,6 +425,34 @@ public final class SocketIOClient: NSObject, SocketEngineClient {
     }
     
     /**
+    Adds a single-use handler for an event.
+    */
+    public func once(event: String, callback: NormalCallback) {
+        Logger.log("Adding once handler for event: %@", type: logType, args: event)
+        
+        let handler = SocketEventHandler(event: event) { data: NSArray?, ack: AckEmitter? in
+            handlers = ContiguousArray(handlers.filter { $0 != handler })
```

Comment:
No, `$0` won't outlive this declaration so there's no need to not make it `strong`.